### PR TITLE
Add raspbian and aarch64 support

### PR DESCRIPTION
**Pull request by auscompgeek**

### CODE REVIEW

File: robotpy_build/platforms.py [hunk 1]
```diff
@@ -51,6 +50,20 @@ def get_platform() -> WPILibMavenPlatform:
     #       be useful to note for the future.
 
     pyplatform = _get_platform()
+
+    if pyplatform == "linux-armv7l":
+        try:
+            import distro
+
+            if distro.id() == "nilrt":
```
Comment (virtuald) [inline robotpy_build/platforms.py]: I don't really like this, but I don't have a good alternative.

Comment (auscompgeek) [reply robotpy_build/platforms.py]: Nor do I honestly. Any alternatives I could come up with will probably be worse.

At least this is mockable to some extent.

**Comment by auscompgeek** [comment]
I guess this might make cross-compiling a little bit harder... I guess I could make `$_PYTHON_HOST_PLATFORM = athena` work, if that works for you @virtuald?

**Comment by virtuald** [comment]
I guess I should decide what to do about this, because of #77

**Pull request merged**